compiler/rustc_abi/src/lib.rs

```diff
@@ -742,11 +742,21 @@ impl Align {
         1 << self.pow2
     }
 
+    #[inline]
+    pub fn bytes_usize(self) -> usize {
+        self.bytes().try_into().unwrap()
+    }
+
     #[inline]
     pub fn bits(self) -> u64 {
         self.bytes() * 8
     }
 
+    #[inline]
+    pub fn bits_usize(self) -> usize {
```

Comment:
Not sure how I feel about adding as yet unused public methods but if you plan on using it soon(tm) in miri, then I'd say it's fine.

Reply:
FWIW I don't plan to add bits_usize, but it exists for Size and I think having a somewhat symmetric API surface here is valuable. It's not always good to remove a function just because it is not used right now, in a large codebase like ours that constantly changes.